calendarext/src/main/java/com/ltman/calendarext/CalendarExt.kt

```diff
@@ -47,11 +45,16 @@ object CalendarExt {
     }
 }
 
-fun Calendar.todayString(context: Context): String {
-    return this.todayString(context, CalendarExt.currentTime())
+/**
+ * use TodayPattern as date time pattern for convert date to string
```

Comment:
Please give example in the comment
e.g. Just now | X minutes ago | X hours ago | Today at H:mm | Yesterday at H:mm | d MMM at H:mm
```kotlin
/**
 * Convert Calendar to String
 * Just now | X minutes ago | X hours ago | Today at H:mm | Yesterday at H:mm | d MMM at H:mm
 **/
```